sycl/source/backend/opencl.cpp

```diff
@@ -44,6 +46,66 @@ __SYCL_EXPORT queue make_queue(const context &Context,
   return detail::make_queue(NativeHandle, Context, nullptr, false,
                             ContextImpl->get_async_handler(), backend::opencl);
 }
+
+// Free functions to query OpenCL backend extensions
+// TODO: Extensions have been deprecated for aspects
+__SYCL_EXPORT bool has_extension(const sycl::platform &SyclPlatform,
+                                 const std::string &Extension) {
+  if (SyclPlatform.get_backend() != sycl::backend::opencl) {
+    throw sycl::exception(
+        errc::backend_mismatch,
+        "has_extension can only be used with an OpenCL backend");
+  }
+
+  auto PlatformImpl = getSyclObjImpl(SyclPlatform);
+  RT::PiPlatform PluginPlatform = PlatformImpl->getHandleRef();
+  plugin Plugin = PlatformImpl->getPlugin();
+
+  // Manual invocation of plugin API to avoid using deprecated
+  // info::platform::extensions call
+  size_t ResultSize = 0;
+  Plugin.call<PiApiKind::piPlatformGetInfo>(
+      PluginPlatform, PI_PLATFORM_INFO_EXTENSIONS, 0, nullptr, &ResultSize);
+  if (ResultSize == 0) {
+    return false;
+  }
```

Comment:
Technically, this goes against our coding standards: [Don’t Use Braces on Simple Single-Statement Bodies of if/else/loop Statements](https://llvm.org/docs/CodingStandards.html#don-t-use-braces-on-simple-single-statement-bodies-of-if-else-loop-statements)

Reply:
I will review this document and make sure my code better adheres to the standards.